client/src/app/editor/file-outputs/file-outputs-table.component.scss

```diff
@@ -41,6 +41,13 @@ mat-progress-spinner {
   height: 16px !important;
 }
 
-.no-files-placeholder {
+.message {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
   margin-top: 1rem;
+
+  ::ng-deep mat-progress-spinner {
```

Comment:
Is `::ng-deep` not going to be/ is deprecated?

Reply:
Actually, I'm not so sure about that. `/deep/` is the selector that's been officially deprecated, while `:ng-deep` is deprecated as well, it's prolly gonna stay as it's the only (angular) way to cross shadow dom boundaries for styles.